### PR TITLE
Review user doc outdoor

### PR DESCRIPTION
**Pull request by camillemonchicourt**
A few typos and adds in Outdoor user manual.
I also suggest to move this section after trek management and valuation modules which are more used.
I also renamed it as these are also management modules

### CODE REVIEW

**Review by submarcos — APPROVED** [review]

File: docs/user-manual.rst [hunk 1]
```diff
@@ -315,6 +248,92 @@ Pour l'ordre d'affichage des catégorie Randonnées, Itinérance et Evènements
...
+Des liens permettent d'ajouter des sites ou parcours en les positionnant directement dans l'arborescence.
+
+**Nomenclatures**
+
+En déroulant le menu en haut à droite de l'écran et en cliquant sur « Admin » il est possible de modifier les nomenclatures.
+
+* Filières : elles servent à regrouper les pratiques pour pouvoir filtrer rapidement les sites ou parcours.
+  Par exemple la filière « eau vive » peut regrouper « kayak » et « canyoning ».
+* Pratiques : les pratiques sportives. Vous pouvez préciser à quelle filière elle appartient.
+* Types de sites : ces catégories permettent d'étiquetter et de filtrer les sites. Elles sont spécifiques à chaque pratique.
```
Comment (numahell) [inline docs/user-manual.rst:298]: Une petite erreur d'orthographe

```suggestion
* Types de sites : ces catégories permettent d'étiqueter et de filtrer les sites. Elles sont spécifiques à chaque pratique.
```

File: docs/user-manual.rst [hunk 2]
```diff
@@ -315,6 +248,92 @@ Pour l'ordre d'affichage des catégorie Randonnées, Itinérance et Evènements
...
+  Par exemple « Site école » pour l'escalade.
+* Échelle de cotation : permet de regrouper les cotations faisant partie de la même échelle. Elles sont spécifiques à chaque pratique.
+
+**Escalade** :
+
+La notion de site est naturelle. Elle peut être définie géographiquement par un polygone.
+Il est possible mais pas obligatoire de créer des sous-site pour des secteurs.
```
Comment (numahell) [inline docs/user-manual.rst:305]: Orthographe et tournure de phrase
```suggestion
Il est possible (mais pas obligatoire) de créer des sous-sites pour représenter des secteurs.
```

**Review by numahell — COMMENTED** [review]
J'ai fait quelques suggestions typographiques

**Comment by camillemonchicourt** [comment]
Merci

File: docs/user-manual.rst [hunk 3]
```diff
@@ -315,6 +248,92 @@ Pour l'ordre d'affichage des catégorie Randonnées, Itinérance et Evènements
...
+
+Geotrek-admin dédie 2 modules aux activités outdoor : les sites et les parcours. Un site correspond à une zone ou à un réseau hydrographique
+à gérer et/ou à valoriser d'un seul tenant : site d'escalade, cours d'eau, zone de vol libre…
+
+Les sites peuvent être subdivisés en sous-sites (dits enfants), qui peuvent eux-mêmes être subdivisés en sous-sites :
+secteurs d'escalade (groupe de blocs ou falaise), aires de décollage ou d'atterrissage…
+
+Chaque site (ou sous-site) peut contenir différents parcours : voie d'escalade, parcours d'eau vive…
+
+Il est possible de regrouper un enchainement de parcours sous forme d'un parcours particulier appelle itinérance :
```
Comment (Chatewgne) [inline docs/user-manual.rst:264]:
```suggestion
Il est possible de regrouper un enchaînement de parcours sous forme d'un parcours particulier appellé itinérance :
```

**Review by Chatewgne — COMMENTED** [review]
Suggestion accents

File: docs/user-manual.rst (hunk 3 above)
Comment (submarcos) [reply docs/user-manual.rst]: "apellé plutot ?

File: docs/user-manual.rst [hunk 4]
```diff
@@ -315,6 +248,92 @@ Pour l'ordre d'affichage des catégorie Randonnées, Itinérance et Evènements
...
+
+Geotrek-admin dédie 2 modules aux activités outdoor : les sites et les parcours. Un site correspond à une zone ou à un réseau hydrographique
+à gérer et/ou à valoriser d'un seul tenant : site d'escalade, cours d'eau, zone de vol libre…
+
+Les sites peuvent être subdivisés en sous-sites (dits enfants), qui peuvent eux-mêmes être subdivisés en sous-sites :
+secteurs d'escalade (groupe de blocs ou falaise), aires de décollage ou d'atterrissage…
+
+Chaque site (ou sous-site) peut contenir différents parcours : voie d'escalade, parcours d'eau vive…
+
+Il est possible de regrouper un enchaînement de parcours sous forme d'un parcours particulier appellé itinérance :
```
Comment (submarcos) [inline docs/user-manual.rst:264]:
```suggestion
Il est possible de regrouper un enchaînement de parcours sous forme d'un parcours particulier appelé itinérance :
```

Comment (camillemonchicourt) [reply docs/user-manual.rst:264]: Oui exact ! Ça me fait révisé mes conjugaisons tout ça :-D

**Pull request merged**